test/jdk/java/awt/Desktop/JMenuItemSetAcceleratorTest.java

```diff
@@ -0,0 +1,109 @@
+/* Copyright (c) 2016, 2022, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import java.awt.Desktop;
+import java.awt.Desktop.Action;
+import java.awt.EventQueue;
+import java.awt.FlowLayout;
+import java.awt.Robot;
+import java.awt.event.KeyEvent;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+import java.awt.event.ActionEvent;
+import javax.swing.SwingUtilities;
+import javax.swing.KeyStroke;
+import javax.swing.JFrame;
+import javax.swing.JMenu;
+import javax.swing.JMenuBar;
+import javax.swing.JMenuItem;
+
+/*
+ * @test
+ * @key headful
+ * @bug 8296275
+ * @summary To verify the setAccelerator method of JMenuItem.
+ * @requires (os.family=="mac")
+ * @run main JMenuItemSetAcceleratorTest
+ */
+
+public class JMenuItemSetAcceleratorTest {
+    private static JFrame frame;
+    private final static CountDownLatch actionPerformLatch =
+        new CountDownLatch(1);
+    private static Robot robot;
+
+    private static void createAndShow() {
+        frame = new JFrame("JMenuItem.setAccelerator");
+        frame.setLayout(new FlowLayout());
+
+        JMenuBar bar = new JMenuBar();
+        JMenu menu = new JMenu("File");
+        JMenuItem menuItem = new JMenuItem("Menu Item");
+
+        menuItem.setAccelerator(
+            KeyStroke.getKeyStroke(KeyEvent.VK_M, ActionEvent.META_MASK));
+        menuItem.addActionListener(e -> {
+            System.out.println("menu item action.");
+            actionPerformLatch.countDown();
+        });
+
+        menu.add(menuItem);
+        bar.add(menu);
+
+        frame.setJMenuBar(bar);
+        frame.pack();
+        frame.setLocationRelativeTo(null);
+        frame.setVisible(true);
+    }
+
+    public static void main(String args[]) throws Exception {
```

Comment:
```suggestion
    public static void main(String[] args) throws Exception {
```

Please use Java-style array declaration.